src/js/components/Tag/Tag.js

```diff
@@ -66,6 +69,7 @@ const Tag = forwardRef(
         {onRemove && (
           <StyledRemoveButton
             onClick={onRemove}
+            aria-label={ariaLabel || 'Remove tag'}
```

Comment:
If we add "Remove tag" as a string, we should include it in default.json message and allow for internationalization.

Reply:
I also wonder if instead of "Remove tag" as the fallback if it should be `Remove ${name} ${value}`

Reply:
good thinking yeah I can do that

Reply:
Note that to properly localize this it would be no `$` in default.json and the format call would be passed the values:
In default.json
```json
"tag": {
  "removeLabel": "Remove {name} {value}"
},
```
And in Tag.js something like:
```jsx
format({
  id: 'tag.removeLabel',
  messages,
  values: {
    name,
    value
  },
})
```
And you have to document it for Grommet's index.d.ts/prop-types.

Typically we've also added a messages prop to the component so that it can be locally set/overridden (not just globally set in Grommet.) Not sure that's always required.